.azure-pipelines/baseline_test/baseline.test.template.yml

```diff
@@ -1,19 +1,11 @@
-variables:
-- group: SONiC-Elastictest
-- name: inventory
-  value: veos_vtb
-- name: testbed_file
-  value: vtestbed.yaml
-- group: GIT_SECRETS
-
 jobs:
 - job: t0_elastictest
   displayName: "kvmtest-t0 by Elastictest"
   timeoutInMinutes: 240
   continueOnError: false
   pool: ubuntu-20.04
   strategy:
-    parallel: $(BASELINE_MGMT_PUBLIC_MASTER_TEST_NUM)
+    parallel: $[ variables['BASELINE_MGMT_PUBLIC_MASTER_TEST_NUM'] ]
```

Comment:
I wonder where is the definition of this variable `BASELINE_MGMT_PUBLIC_MASTER_TEST_NUM` ?

Reply:
Yes, and parallel variable type should be number, but our env imported would be string